### PR TITLE
Small updates

### PR DESCRIPTION
**Pull request by fomdt**

### CODE REVIEW

File: docs/developer-docs/devquests.md [hunk 1]
```diff
@@ -63,11 +63,11 @@ DevQuests are achievement development-related quests that Developers can take pa
...
 **Repeatable**: No
 
-1. This is a chance to tighten the code up, do not just add a bad line of code
-2. You do not need a revision vote to tighten
+1. This is a chance to tighten the code up, do not just add a bad line of code.
+2. You do not need a revision vote to tighten.
 3. Achievements cannot be your own, you should just fix these anyway!
-4. You must Comments on each achievement or in the forum on what you changed for log purposes
-5. May not overlap with the required repair plan for DevQuest 002 or with any completion requirements for DevQuest 013
+4. You must Comments on each achievement or in the forum on what you changed for log purposes.
```
Comment (wescopeland) [inline docs/developer-docs/devquests.md]:
```suggestion
4. You must comment on each achievement or in the forum on what you changed for log purposes.
```

File: docs/developer-docs/devquests.md [hunk 2]
```diff
@@ -63,11 +63,11 @@ DevQuests are achievement development-related quests that Developers can take pa
 **Goal**: Tighten up the logic for 20 single-condition achievements  
 **Repeatable**: No
 
-1. This is a chance to tighten the code up, do not just add a bad line of code
-2. You do not need a revision vote to tighten
+1. This is a chance to tighten the code up, do not just add a bad line of code.
```
Comment (wescopeland) [inline docs/developer-docs/devquests.md]:
```suggestion
1. This is a chance to tighten the code up. Do not just add a bad line of code.
```

File: docs/developer-docs/devquests.md [hunk 3]
```diff
@@ -46,9 +46,9 @@ DevQuests are achievement development-related quests that Developers can take pa
 **Goal**: Repair 3 sets from provided batch  
 **Repeatable**: Yes (Up to 3 times)
 
-1. Pick a game from the list of approved DQ2 sets and then post your claim to the [`#devquest`](https://discord.com/channels/310192285306454017/842845740078334002) channel on Discord or send to DevQuest inbox
+1. Pick a game from the list of approved DQ2 sets and then post your claim to the [`#devquest`](https://discord.com/channels/310192285306454017/842845740078334002) channel on Discord or send to DevQuest inbox.
 2. Follow the revision process as normal: Post the plan on the forum, make a revision vote if any changes aren't on the "no need for approval list," and make a revision claim if adding achievements.
-3. Resolve any open tickets for achievements kept in the set, Tickets for achievements being left demoted can be simply closed with reason of demotion cited
+3. Resolve any open tickets for achievements kept in the set, Tickets for achievements being left demoted can be simply closed with reason of demotion cited.
```
Comment (wescopeland) [inline docs/developer-docs/devquests.md]:
```suggestion
3. Resolve any open tickets for achievements kept in the set. Tickets for achievements being left demoted can be simply closed with reason of demotion cited.
```

File: docs/developer-docs/devquests.md [hunk 4]
```diff
@@ -63,11 +63,11 @@ DevQuests are achievement development-related quests that Developers can take pa
 **Goal**: Tighten up the logic for 20 single-condition achievements  
 **Repeatable**: No
 
-1. This is a chance to tighten the code up, do not just add a bad line of code
-2. You do not need a revision vote to tighten
+1. This is a chance to tighten the code up, do not just add a bad line of code.
+2. You do not need a revision vote to tighten.
```
Comment (wescopeland) [inline docs/developer-docs/devquests.md]: Does anyone know what this line means? @suspect15 @redwizard42 ?

File: docs/developer-docs/devquests.md [hunk 5]
```diff
@@ -76,19 +76,21 @@ DevQuests are achievement development-related quests that Developers can take pa
...
+4. At least 85% of sets have dynamic rich presence _(Vet Dev I required)_.
+5. Yielded unlocks have exceeded 50,000.
+6. As a sole developer, submitted one or more sets for code review and received an overall grade of at least 22.
+7. Has used at least 75% of all currently available flags and features _(Vet Dev I and VI required)_.
+8. Has obtained at least two DevQuest badges.
+9. Has obtained at least five badges through DevQuest.
 
--_Note: A constantly evolving toolkit means that these requirements are subject to change at any time_
+::: warning NOTE
+A constantly evolving toolkit means that these requirements are subject to change at any time
```
Comment (wescopeland) [inline docs/developer-docs/devquests.md]:
```suggestion
A constantly evolving toolkit means that these requirements are subject to change at any time.
```

File: docs/developer-docs/devquests.md [hunk 6]
```diff
@@ -315,9 +317,9 @@ You must have developed sets for 10 or more supported platforms before opt-in. O
 
 1. That means both sets must be DIFFERENT SPORTS. You cannot do two baseball games, or two soccer games.
 2. Must be full sets containing all content and challenges appropriate to the game.
-3. No Homebrew/Hacks/Unlicensed
-4. New sets only: No Multi Set or Revisions
-5. Other picks are subject to DQ team approval if eligibility is questionable
+3. No Homebrew/Hacks/Unlicensed.
+4. New sets only: No Multi Set or Revisions.
```
Comment (wescopeland) [inline docs/developer-docs/devquests.md]:
```suggestion
4. New sets only. No multiplayer subsets or revisions.
```
Let's go ahead and change this while we're here. I think we'll wind up rebranding "Multi" to "Multiplayer".

**Review by wescopeland — COMMENTED** [review]
Thanks! A few suggestions below.

File: docs/developer-docs/devquests.md (hunk 4 above)
Comment (wescopeland) [reply docs/developer-docs/devquests.md]: I guess it means "to tighten the code up".

Comment (redwizard42) [inline docs/developer-docs/devquests.md]:
```suggestion
2. You do not need a revision vote to tighten up this kind of logic.
```

**Review by redwizard42 — CHANGES_REQUESTED** [review]
I think it would be helpful to narrow the scope of grammar and format adjustment PRs. Something like "Grammar and Format touch-up for DevQuest event document" is more informative than "Small Updates". Probably can still touch up a few related documents at the same time. but a narrower scope makes the PR's purpose more obvious.

(Unless Wes prefers them to be done in a bigger chunk like this for workload purposes, in which case I'll defer to him ;) )

**Review by redwizard42 — APPROVED** [review]

**Review by wescopeland — APPROVED** [review]

**Pull request merged**